<ui version="4.0" >
 <author>Alexander Saal - alex.saal@gmx.de</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DBoxFE_GameDatabase</class>
 <widget class="QWidget" name="DBoxFE_GameDatabase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gamedatabase</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/dboxfe.qrc" >:/pics/images/default.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="gBoxGames" >
     <property name="title" >
      <string>Games</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="QTreeWidget" name="twGames" >
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="rootIsDecorated" >
         <bool>true</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Gamefile</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Games in database:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnClose" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>Cl&amp;ose</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChange" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="gBoxOption" >
     <property name="title" >
      <string>Option</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblGameName" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Gamename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGameName" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblGameBinary" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Gamebinary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGameBinary" />
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectBin" >
          <property name="minimumSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="icon" >
           <iconset resource="../res/dboxfe.qrc" >:/pics/images/folder-open_16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../res/dboxfe.qrc" />
 </resources>
 <connections/>
</ui>
